dashboards-reports/public/components/report_definitions/report_settings/report_settings.tsx

```diff
@@ -573,6 +608,17 @@ export function ReportSettings(props: ReportSettingProps) {
       .catch((error) => {
         console.log('error when fetching saved searches:', error);
       });
+
+    await httpClientProps
+      .get('/api/notebooks/')
```

Comment:
`../api/notebooks/`? similar to how reporting api is called above

Reply:
Good catch, will change